src/applications/coronavirus-screener/components/MultiQuestionForm.jsx

```diff
@@ -27,7 +27,7 @@ export default function MultiQuestionForm({ questions, defaultOptions }) {
         if (completed === false) {
           recordEvent({
             event: 'covid-screening-tool-result-displayed',
-            'screening-tool-result': formState.result,
```

Comment:
Has formState.status already updated when recordEvent is called?  If not it will send the 'incomplete' status and not one of the expected values.  Also, need to confirm the label values as discussed [here](https://github.com/department-of-veterans-affairs/vets-website/pull/12875#discussion_r435882756). It looks like we will be sending `pass` or `more-screening`, and want to confirm that with @jonwehausen

Reply:
Thanks guys @stephenBarrs and @mchelen-gov 

If we could ensure 1 of the 2 values if passed here that would be great:
- `pass`
- `more-screening`

Reply:
good catch @stephenBarrs, yeah it should `newStatus`